ccd-definition/CaseEvent/MultiState.json

```diff
@@ -166,7 +166,7 @@
     "Name": "Add statement of service (c9)",
     "Description": "Add statement of service",
     "DisplayOrder": 13,
-    "PreConditionState(s)": "Submitted;CLOSED",
+    "PreConditionState(s)": "Submitted;Gatekeeping;PREPARE_FOR_HEARING;CLOSED",
```

Comment:
same here addStatementOfServiceGatekeeping has CRU for LASOLICITOR

Reply:
updated